File: fixtures/adapter-go/alias_import_bug/mappers/deep/view_model_mapper.go
package deep

import (
    models "github.com/test/app/models"
)

type ViewModelMapper struct{}

func (m *ViewModelMapper) Map(data *models.UserPayload) map[string]interface{} {
    return map[string]interface{}{
        "id": data.UserID,
        "ts": data.Timestamp,
        "st": data.Status,
    }
}

File: fixtures/adapter-rust/alias_import_bug/src/mappers/view_model_mapper.rs
use crate::models::user_payload::UserPayload as ResponseModel;

pub struct ViewModelMapper;

impl ViewModelMapper {
    pub fn map(&self, data: &ResponseModel) {
        let _ = &data.user_id;
        let _ = &data.timestamp;
        let _ = &data.status;
    }
}
